### ISSUE TITLE
Getting Started documentation does not work

### ISSUE THREAD

**Issue by dunlapm** [SharePoint/sp-dev-docs#7296]
I have tried repeatedly to follow the documentation for setting up a local dev environment and building the "hello world" web part. Everything seems to work as expected up until I run `gulp serve` and the workbench doesn't load. First it appeared that `@microsoft/sp-webpart-workbench` was not installed. 

I tried the same exercise in WSL 2 - Ubuntu, and the project was able to start as expected.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6c25a90e-829c-b436-d8ab-f9c010588ee1
* Version Independent ID: 768fa9a1-2375-fb5a-be58-cb66320cc789
* Content: [Build your first SharePoint client-side web part (Hello World part 1)](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/get-started/build-a-hello-world-web-part)
* Content Source: [docs/spfx/web-parts/get-started/build-a-hello-world-web-part.md](https://github.com/SharePoint/sp-dev-docs/blob/master/docs/spfx/web-parts/get-started/build-a-hello-world-web-part.md)
* Product: **sharepoint**
* Technology: **sharepoint-framework**
* GitHub Login: @VesaJuvonen
* Microsoft Alias: **vesaj**

**Comment by bcameron1231** [comment]
Can you provide us with the version of SPFx you are using, as well as any helpful information such as any errors that are popping up for you.

**Comment by dunlapm** [comment]
It looks like this issue was caused by a transient error while the `yo @microsoft/sharepoint` command was being run. I am not able to recreate, and running the yeoman generator on a different machine behaved as expected. I should note that I did not see any obvious errors the first time running the generator which did not make me consider that something had failed to download/install during that script run.
